resources/definitions/fdmextruder.def.json

```diff
@@ -120,6 +131,18 @@
                     "settable_per_meshgroup": false,
                     "type": "str"
                 },
+                "machine_extruder_change_duration":
+                {
+                    "default_value": 0,
+                    "description": "The time it'll take for the extruder change to execute.",
```

Comment:
Same as the other one. The description should be a bit longer to explain why and when one would use this.